pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -404,6 +404,7 @@ func (a *FlavorAssigner) findFlavorForPodSetResource(
 	assignedFlavorIdx := -1
 	idx := a.wl.LastAssignment.NextFlavorToTryForPodSetResource(psID, resName)
 	for ; idx < len(resourceGroup.Flavors); idx++ {
+		assignedFlavorIdx = idx
```

Comment:
Can you please cover this change with a unit or integration test?

Reply:
also we should probably rename the variable to `attemptedFlavorIdx`, but that can be done in a follow up.